Kulipa.Sdk/Models/Requests/Users/UserWallet.cs

```diff
@@ -14,6 +15,7 @@ public sealed record UserWallet
         /// </summary>
         [Required]
         [JsonPropertyName("blockchain")]
+        [JsonConverter(typeof(KebabCaseLowerJsonStringEnumConverter))]
```

Comment:
Why do we use kebab here? What are the exceptional cases to use kebab and the other one?